src/UpdatedPackagesCollector.js

```diff
@@ -150,9 +150,9 @@ export default class UpdatedPackagesCollector {
       return file.replace(folder + path.sep, "");
     });
 
-    if (this.flags.ignore) {
+    if (this.flags.publishConfig && this.flags.publishConfig.ignore) {
```

Comment:
This shouldn't be necessary.  `this.flags` in the context of the `UpdatedPackagesCollector` comes from the command's `.getOptions()`, which should pull in values from `publishConfig` automatically.

Is this failing to pick up the option in some context?